dolphinscheduler-ui/src/views/projects/workflow/definition/components/start-modal.tsx

```diff
@@ -333,7 +333,7 @@ export default defineComponent({
               v-model:value={this.startForm.warningGroupId}
               clearable
             />
-          </NFormItem>
+          </NFormItem>) : null}
```

Comment:
Maybe you can try the logical and(&&) operator.

Reply:
@Amy0104 done